prime-router/docs/api/waters-delivery.yml

```diff
@@ -30,6 +30,26 @@ paths:
           required: true
           schema:
             type: string
+        - name: reportId
+          in: query
+          description: the reportId within the deliveries of the organization and client
+          schema:
+            type: string
+        - name: fileName
+          in: query
+          description: the fileName within the deliveries of the organization and client
+          schema:
+            type: string
+        - name: receivingOrgSvcStatus
+          in: query
+          description: "Customer status of the service used by the organization."
```

Comment:
Nit: maybe a small update here indicating multiple values can be passed here

Reply:
Will add